src/main/java/vn/unigap/api/entity/Employer.java

```diff
@@ -60,14 +66,18 @@ public void setName(String name) {
         this.name = name;
     }
 
-    public int getProvince() {
-        return province;
+    public int getProvinceId() {
+        return province.getId();
     }
 
-    public void setProvince(int province) {
+    public void setProvince(Province province) {
```

Comment:
you can use @Data annotation on top class , you will not declare function setter, getter.

Reply:
Thanks.

Reply:
I did change it. Please help me to review! TIA!